packages/deployment/app/cumulus_api.template.yml

```diff
@@ -64,6 +64,17 @@ Resources:
         Types:
           - 'PRIVATE'
       {{/ifPrivateApi}}
+      # Needed for deploy, but in an NGAP account, this will get overwritten
+      Policy:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: "Allow"
+            Principal:
+              AWS:
+                Fn::Sub: "arn:aws:iam::${AWS::AccountId}:root"
+            Action:
+              - "execute-api:Invoke"
+            Resource: "execute-api:/*"
```

Comment:
I realize it's a dummy, but is this `Resource` accurate? Also, should we wrap the whole policy in the `{{# ifPrivateApi }}`? Lack of resource policy seems to come up on private RestApis only.

Reply:
In regards to the `Resource` comment, should this be something like: `arn:aws:execute-api:${region}:${account-id}:${api-id}/${stage}`? Referencing the (Control Who Can Call an API Gateway API Method with IAM Policies)[https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-control-access-using-iam-policies-to-invoke-api.html]

Reply:
@mennovandiermen yeah i think i should put it in the `if`. What do you think the `Resource` should be? Reading through docs I'd think that would work.

Reply:
Jacob's suggestion looks fine to me.